Source/Add-ins/Profile.Common/MilSpaceProfileCalsController.cs

```diff
@@ -1176,10 +1179,27 @@ private IPoint GetPointFromGeoCalculator()
             return null;
         }
 
-        //private IPoint GetPointFromObservationPoints()
-        //{
+        private IPoint GetPointFromObservationPoints()
+        {
+            var observPointsLayer = _mapLayersManager.FindFeatureLayer("MilSp_Visible_ObservPoints");
+
```

Comment:
The layer should be taken form the corresponding module, using module interaction